internal/pkg/exec/docker.go

```diff
@@ -159,3 +161,38 @@ func imageName(uri, tag string) string {
 	}
 	return fmt.Sprintf("%s:%s", uri, tag)
 }
+
+// GetHelperProviderFromDockerCfg will fetch credential store used by docker from config file
+func (c DockerCommand) GetHelperProviderFromDockerCfg() (string) {
+	// Look into the default locations
+	pathsToTry := []string{"/.docker/config.json", ".dockercfg"}
+	for _, path := range pathsToTry {
+		content, err := ioutil.ReadFile(fmt.Sprintf("%s/%s",os.Getenv("HOME"), path))
```

Comment:
Will this result in a double "//" such as "<homepath>//.docker/config.json"?
```suggestion
	pathsToTry := []string{filepath.Join(".docker, config.json"), ".dockercfg"}
	for _, path := range pathsToTry {
		content, err := ioutil.ReadFile(filepath.Join(os.Getenv("HOME"), path))
```